### DISCUSSION TITLE
Kiwi browser builds

### DISCUSSION THREAD

**Discussion by Achyuth072** [ImranR98/Obtainium#1713] [Q&A] [answered]
I've been trying to get the arm64 dev updates for the browser but I can't figure it out.
https://github.com/kiwibrowser/src.next/releases/tag/8822657649
How do I ads just the dev builds?

**Comment by DwainZwerg** [comment, marked as answer]
Add "com.kiwibrowser.browser.dev-arm64" to the line "Filter APKs by Regular Expression"; the configuration will then look like this: [config](https://apps.obtainium.imranr.dev/redirect?r=obtainium://app/%7B%22id%22%3A%22com.kiwibrowser.browser.dev%22%2C%22url%22%3A%22https%3A%2F%2Fgithub.com%2Fkiwibrowser%2Fsrc.next%22%2C%22author%22%3A%22kiwibrowser%22%2C%22name%22%3A%22src.next%22%2C%22preferredApkIndex%22%3A0%2C%22additionalSettings%22%3A%22%7B%5C%22includePrereleases%5C%22%3Atrue%2C%5C%22fallbackToOlderReleases%5C%22%3Atrue%2C%5C%22filterReleaseTitlesByRegEx%5C%22%3A%5C%22%5C%22%2C%5C%22filterReleaseNotesByRegEx%5C%22%3A%5C%22%5C%22%2C%5C%22verifyLatestTag%5C%22%3Afalse%2C%5C%22dontSortReleasesList%5C%22%3Afalse%2C%5C%22useLatestAssetDateAsReleaseDate%5C%22%3Afalse%2C%5C%22trackOnly%5C%22%3Afalse%2C%5C%22versionExtractionRegEx%5C%22%3A%5C%22%5C%22%2C%5C%22matchGroupToUse%5C%22%3A%5C%22%5C%22%2C%5C%22versionDetection%5C%22%3Atrue%2C%5C%22releaseDateAsVersion%5C%22%3Afalse%2C%5C%22useVersionCodeAsOSVersion%5C%22%3Afalse%2C%5C%22apkFilterRegEx%5C%22%3A%5C%22com.kiwibrowser.browser.dev-arm64%5C%22%2C%5C%22invertAPKFilter%5C%22%3Afalse%2C%5C%22autoApkFilterByArch%5C%22%3Atrue%2C%5C%22appName%5C%22%3A%5C%22Kiwi%20Browser%20Dev%5C%22%2C%5C%22shizukuPretendToBeGooglePlay%5C%22%3Afalse%2C%5C%22exemptFromBackgroundUpdates%5C%22%3Afalse%2C%5C%22skipUpdateNotifications%5C%22%3Afalse%2C%5C%22about%5C%22%3A%5C%22%5C%22%7D%22%2C%22overrideSource%22%3Anull%7D)

**Reply by Achyuth072** [reply]
Thanks it worked.

**Reply by DwainZwerg** [reply]
@Achyuth072 Can you please close this discussion as resolved?
